.template/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```

Comment:
Why do we upgrade the version here? 🤔

Reply:
@manh-t @doannimble From this PR https://github.com/nimblehq/flutter_templates/pull/71/files, I think we should update this version to `10.0` as well. We might don't need this to run the app locally but release https://docs.flutter.dev/deployment/ios#updating-the-apps-deployment-version

Reply:
@luongvo When I run the app on iOS device, it auto-upgrades the version. Yeah, I will upgrade to `10.0` to match the requirement 🙏

Reply:
What we did in the last project as well 

![image](https://user-images.githubusercontent.com/16315358/163544710-8419cee7-34b1-45a6-82aa-81473c4c129a.png)